### PR TITLE
Srishtirupa main

### PR DESCRIPTION
**Pull request by Srishtirupa3**

### CODE REVIEW

File: Comments_Module.py [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+        Exception: If any other unexpected exception occurs.
+
+    Usage:
+        - Send a POST request to the 'add_projectcomment' endpoint.
+        - The request data must be in JSON format and include the following fields:
+            - 'project_id' (integer): The ID of the project to add the comment to.
+            - 'user_id' (integer): The ID of the user adding the comment.
+            - 'description' (string): The content of the comment to be added.
+"""
+    try:
```
Comment (pratiksatham) [inline Comments_Module.py:41]: make a short doc string

File: Comments_Module.py [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+        KeyError: If the required field 'issue_id' is missing in the request data.
+        mysql.connector.Error: If there is an error related to the MySQL database.
+        Exception: If any other unexpected exception occurs.
+
+    Usage:
+        - Send a POST request to the 'display_issuewisecomments' endpoint.
+        - The request data must be in JSON format and include the following field:
+            - 'issue_id' (integer): The ID of the issue for which to retrieve the comments.
+    """
+
```
Comment (pratiksatham) [inline Comments_Module.py:103]: Write a code in pep 8 format

File: test_for_updateprojectwisecomments.py [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+        # Get the response content as JSON
+        Response_msg = r.json()
+        # Print the JSON response
+        print(Response_msg)
+        print("case2")
+        self.assertEqual(r.status_code,400,msg="Response code doesn't match") 
+        self.assertEqual(Response_msg,{"error": "Missing 'project_id' in request data"},msg="Expected output doesn't match the resultant output.")
+
+#############################################
+
```
Comment (pratiksatham) [inline test_for_updateprojectwisecomments.py:57]: remove all the hashtag lines

File: test_for_updateprojectwisecomments.py [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+        Response_msg = r.json()
+        # Print the JSON response
+        print(Response_msg)
+        print("case5")
+        self.assertEqual(r.status_code,400,msg="Response code doesn't match") 
+        self.assertEqual(Response_msg,{"error": "Missing 'comment_id' in request data"},msg="Expected output doesn't match the resultant output.")
+
+
+#############################################
+
```
Comment (pratiksatham) [inline test_for_updateprojectwisecomments.py:135]: remove the hash lines

**Review by pratiksatham — APPROVED** [review]

File: Comments_Module.py (hunk 1 above)
Comment (Srishtirupa3) [reply Comments_Module.py:41]: For proper documentation this much doc string is required.

File: Comments_Module.py (hunk 2 above)
Comment (Srishtirupa3) [reply Comments_Module.py:103]: Its in pep8 format only,but can you plz give the point if i missed something.

File: test_for_updateprojectwisecomments.py (hunk 3 above)
Comment (Srishtirupa3) [reply test_for_updateprojectwisecomments.py:57]: Its for better code readability and segregation.

File: test_for_updateprojectwisecomments.py (hunk 4 above)
Comment (Srishtirupa3) [reply test_for_updateprojectwisecomments.py:135]: its for better code readability and segregation.

**Review by Simran2711 — APPROVED** [review]

**Review by Chandrashekhar-Bhavsar — APPROVED** [review]

**Pull request merged**